check-submodule.py

```diff
@@ -158,8 +180,14 @@ def dump_minimal_tests(
     filename = code_spans_object["name"]
 
     # Read code spans from the input source code
-    with open(filename, "r") as input_file:
-        lines = input_file.readlines()
+    with open(filename, "r", encoding="utf-8", errors="ignore") as input_file:
+        try:
+            # Decode only utf-8 source code files
+            lines = input_file.readlines()
+        except UnicodeDecodeError:
```

Comment:
This should no longer happen now that you're ignoring errors, right?

Reply:
Forgot to remove those lines :P 

Thanks! :)